src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/DittoBaseConverter.cs

```diff
@@ -0,0 +1,175 @@
+﻿namespace Our.Umbraco.Ditto
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Globalization;
+    using System.IO;
+    using System.Xml;
+
+    using global::Umbraco.Core;
+    using global::Umbraco.Core.Models;
+    using global::Umbraco.Web;
+
+    /// <summary>
+    /// The Ditto base converter containing reusable properties and converting methods for <see cref="IPublishedContent"/> instances.
+    /// All other Ditto converters should inherit from this class.
+    /// </summary>
+    public abstract class DittoBaseConverter : TypeConverter
+    {
+        /// <summary>
+        /// Gets the <see cref="UmbracoHelper"/> for querying published content or media.
+        /// </summary>
+        public static UmbracoHelper UmbracoHelper
```

Comment:
General question, is it better to use a `static` property or for it to be part of the instance?